README.md

```diff
@@ -101,3 +115,57 @@ Do:
       - '// '
       - ''
 ```
+
+### markdown-toc
+
+Generates a [Prettier](https://pretter.io)-compatible table of contents for
+Markdown files.
+
+In a markdown file, put the `<!-- toc -->` and `<!-- tocstop -->` markers to
+indicate where to put the table of contents:
+
+```md
+# Document title
+
+<!-- toc -->
+<!-- tocstop -->
+
+## Header
+
+### Sub-header
+```
+
+This will generate a table of contents based on non-title headers:
+
+```md
+# Document title
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Header](#header)
+    -   [Sub-header](#sub-header)
+
+<!-- toc -->
+
+## Header
+
+### Sub-header
+```
+
+In `.pre-commit-config.yaml`, put:
+
+```yaml
+- id: markdown-toc
+```
+
+This generates bullets with a four space indent. When used with Prettier, it's
```

Comment:
FWIW, the new text here is exactly what I was looking for to make the 4-space indent assumption explicit. Thanks. (And sorry I didn't see it when responding to the other comment thread.)

Reply:
Acknowledged